filters/filter.go

```diff
@@ -37,25 +37,26 @@ func HasRequiredRoles(viewRoles []string) beego.FilterFunc {
 			payload,
 			fmt.Sprintf("https://%s/claims/roles", conf.App.Domain),
```

Comment:
What type would tokenRoles be here? Again would this be a string or some array of strings within payload?